cudamapper/src/cudamapper_utils.cpp

```diff
@@ -65,5 +66,58 @@ void fuse_overlaps(std::vector<Overlap>& fused_overlaps, const std::vector<Overl
     set_relative_strand(fused_overlap);
     fused_overlaps.push_back(fused_overlap);
 }
+
+std::vector<std::string> kmerize_string(const std::string& s, int kmer_length, int stride)
```

Comment:
I agree - uint8_t would probably do, but should I go ahead and make it a uint32_t?

Reply:
@ahehn-nv and I discussed this a few time and decided to go for singed types. I don't think we changed it everywhere already, but new new codes should definitely use it. So in this case I'd use `std::int32_t`. I think there are still some places where we keep them 64-bit. if you get compiler warnings simply stick with 64-bit.
I'm planning to add kmer and window size types to `types.hpp` to avoid this type of confusion

Reply:
resolved by 100bb80